src/components/MessageCommerce.js

```diff
@@ -86,10 +86,32 @@ class MessageCommerce extends PureComponent {
     onMentionsClickMessage: PropTypes.func,
     /** Position of message in group. Possible values: top, bottom, middle, single */
     groupStyles: PropTypes.array,
+    /**
+     * The component that will be rendered if the message has been deleted.
+     * All of Message's props are passed into this component.
+     */
+    MessageDeleted: PropTypes.func,
   };
 
   static defaultProps = {
     Attachment,
+    MessageDeleted: ({ message, t, isMyMessage }) => {
+      const messageClasses = isMyMessage(message)
+        ? 'str-chat__message-commerce str-chat__message-commerce--left'
+        : 'str-chat__message-commerce str-chat__message-commerce--right';
+
+      return (
+        <React.Fragment>
+          <span
+            key={message.id}
+            className={`${messageClasses} str-chat__message--deleted`}
+          >
+            {t('This message was deleted...')}
+          </span>
+          <div className="clearfix" />
+        </React.Fragment>
+      );
+    },
```

Comment:
Probably good to split this out into a separate component

Reply:
I split this out, even though I realized while doing so that the block of code for the deleted message in `MessageCommerce` was redundant, and it would actually always return null if a message had been deleted in what's currently on master.

For now, I've made `MessageDeleted` a separate component, even though it's only used in `MessageSimple`. I think it makes sense to keep `MessageSimple`. a bit smaller, but let me know if you prefer it to stay inside `MessageSimple`.